src/Search.php

```diff
@@ -9089,6 +9089,8 @@ public static function makeTextSearchValue($val)
         // 2. Escape raw value to protect SQL special chars.
         $val = Sanitizer::dbEscape(Sanitizer::unsanitize($val));
 
+        $val = str_replace('\\\\', '\\\\\\\\', $val);
```

Comment:
```suggestion
        // Backslashes must be doubled in LIKE clause, according to MySQL documentation:
        // https://dev.mysql.com/doc/refman/8.0/en/string-comparison-functions.html
        // > To search for \, specify it as \\\\; this is because the backslashes are stripped once by the parser
        // > and again when the pattern match is made, leaving a single backslash to be matched against.
        $val = str_replace('\\', '\\\\', $val);
```